OpenTabletDriver.Configurations/Configurations/Huion/Kamvas Pro 19 (4k).json

```
@@ -0,0 +1,42 @@
+{
+    "Name": "Huion Tablet_GT1902",
+    "Specifications": {
+        "Digitizer": {
+            "Width": 409,
+            "Height": 230,
+            "MaxX": 81792,
+            "MaxY": 46006
```

You should not have to guess these, measure the x and y coordinates in the debugger and get the absolute max you can, then divide by the lpmm which in this case is 200 (5080lpi/25.4)

Provided those *are* the maximum values you can get, change them to this as it is important to provide accurate digitizer specs.
```suggestion
            "Width": 408.96,
            "Height": 230.03,
            "MaxX": 81792,
            "MaxY": 46006
```

---

These were indeed the highest I was able to replicate